lib/whitehall_importer/import.rb

```diff
@@ -84,7 +84,7 @@ def check_document_integrity(document)
     def check_edition_integrity(edition)
       integrity_checker = IntegrityChecker.new(edition)
 
-      raise AbortImportError, integrity_checker.problems.join(", ") unless integrity_checker.valid?
+      raise WhitehallImporter::IntegrityCheckError.new(edition.state, integrity_checker.problems, integrity_checker.proposed_payload) unless integrity_checker.valid?
```

Comment:
state doesn't feel the right thing to be passing in here as it can vary into scheduled, removed etc. Aren't we more interested in whether it's live or draft?

Reply:
Yeah, it was another thing on my To do list.